easybuild/easyconfigs/g/GTK+/GTK+-3.22.30-fosscuda-2018b.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'GTK+'
+version = '3.22.30'
+
+homepage = 'https://developer.gnome.org/gtk+/stable/'
+description = """
+ The GTK+ 2 package contains libraries used for creating graphical user interfaces for applications.
+"""
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['a1a4a5c12703d4e1ccda28333b87ff462741dc365131fbc94c218ae81d9a6567']
+
+builddependencies = [
+    ('GObject-Introspection', '1.54.1', '-Python-2.7.15'),
```

Comment:
No clue, I was just taking a copy of existing GTK+ .ebs and they one i used didn't have it.
But 2.24.32-foss-2018b does, so perhaps it should be explicit instead of getting it pulled from GObject...